libraries/incubator/ORM/Entity/EntityBuilder.php

```diff
@@ -399,7 +397,7 @@ public function handleStorage(Element $storage)
 					break;
 			}
 
-			$this->reflector->setStorageProvider(new $handler($param));
```

Comment:
Fixed the test

https://github.com/joomla-projects/joomla-pythagoras/pull/85/commits/8d3ad7553626e707d7217545c024b9b3aa37a018
https://travis-ci.org/joomla-projects/joomla-pythagoras/jobs/127507637